src/module/helpers.ts

```diff
@@ -52,21 +52,20 @@ export class Helpers {
         }
 
         // Some values will have their total overridden directly.
-        if (value.override && value.override.mode === 'replace') {
+        if (value.override) {
             // Still apply a possible value range, even if override says otherwise.
             value.value = Helpers.applyValueRange(value.override.value, options);
             return value.value;
         }
 
         value.value = parts.total + value.base;
 
-        // Some values will apply min / max values to their totals.
-        if (value.override && value.override.mode === 'min') {
-            value.value = Helpers.applyValueRange(value.value, { min: value.override.value });
+        // In case of multiple effect changes, prioritize downgrade as I want the universe to suffer.
+        if (value.downgrade) {
+            value.value = Helpers.applyValueRange(value.value, { max: value.downgrade.value });
+        } else if (value.upgrade) {
```

Comment:
👀 I see an `else if` here… which means if both `downgrade` and `upgrade` exist, only the suffering (`downgrade`) happens, and the glorious whiplash of an immediate `upgrade` is lost.

Please change this to two separate `if`s. That way we can `downgrade` and then `upgrade`, giving the universe the full rollercoaster of despair and false hope it deserves. 🎢✨